pkg/rbd/nodeserver.go

```diff
@@ -303,9 +303,25 @@ func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeS
 	// Publish Path
 	fsType := req.GetVolumeCapability().GetMount().GetFsType()
 	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: mount.NewOsExec()}
+	existingFormat, err := diskMounter.GetDiskFormat(devicePath)
+	if err != nil {
+		return err
+	}
+	if existingFormat == "" && (fsType == "ext4" || fsType == "xfs") {
+		args := []string{devicePath}
+		if fsType == "ext4" {
+			args = []string{"-F", "-m0", "-Enodiscard", devicePath}
+		} else if fsType == "xfs" {
+			args = []string{"-f", "-K", devicePath}
```

Comment:
Let's drop -F/-f.  Testing existingFormat and then forcing mkfs is weird.

Reply:
Done.